azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -296,6 +334,7 @@ File downloadProjectIfNotExists(final ProjectDirectoryMetadata proj, final int e
       return null;
     }
 
+    LOGGER.info("Project {} not cached, downloading.", proj);
```

Comment:
Project will never be cached. Why is this log line added?

Reply:
debugging line :)